### PR TITLE
Get values of multiple fields

### PR DESCRIPTION
**Pull request by David-BMS**
## Status
Ready

## Related Issues
https://github.com/demisto/etc/issues/18549

## Description
This transformer allows to get values from multiple context location at the same time.

## Screenshots
Paste here any images that will help the reviewer

## Required version of Demisto
4.5.0

## Does it break backward compatibility?
   - Yes
       - Further details:
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

**Comment by yaron-libman** [comment]
Reviewed yml file.

File: Scripts/GetValuesOfMultipleFIelds/GetValuesOfMultipleFIelds.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        _list.append(update_with)
+
+
+def main():
+    args: Dict = demisto.args()
+    root = args.get('Key')
+    if root:
+        if not isinstance(root, list):
+            root = [root]
+        keys: List = args.get('List', '').split(',')
```
Comment (yaakovi) [inline Scripts/GetValuesOfMultipleFIelds/GetValuesOfMultipleFIelds.py]:
```suggestion
        keys: List = args.get('list', '').split(',')
```

File: Scripts/GetValuesOfMultipleFIelds/GetValuesOfMultipleFIelds.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+def update_list(_list: List, update_with):
+    if isinstance(update_with, list):
+        _list.extend(update_with)
+    else:
+        _list.append(update_with)
+
+
+def main():
+    args: Dict = demisto.args()
+    root = args.get('Key')
```
Comment (yaakovi) [inline Scripts/GetValuesOfMultipleFIelds/GetValuesOfMultipleFIelds.py]:
```suggestion
    root = args.get('key')
```

File: Scripts/GetValuesOfMultipleFIelds/GetValuesOfMultipleFIelds.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+args:
+- name: Key
```
Comment (yaakovi) [inline Scripts/GetValuesOfMultipleFIelds/GetValuesOfMultipleFIelds.yml]:
```suggestion
- name: key
```

File: Scripts/GetValuesOfMultipleFIelds/GetValuesOfMultipleFIelds.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  required: true
+  description: The base context path under which to find the fields. For example Demisto.Results.
+- name: List
```
Comment (yaakovi) [inline Scripts/GetValuesOfMultipleFIelds/GetValuesOfMultipleFIelds.yml]:
```suggestion
- name: list
```

File: Scripts/GetValuesOfMultipleFIelds/GetValuesOfMultipleFIelds.yml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  required: true
+  description: The list of fields to retrieve from context. The list can contain comma
+    seperated values. For example key1,key1
+  isArray: true
+- name: Value
```
Comment (yaakovi) [inline Scripts/GetValuesOfMultipleFIelds/GetValuesOfMultipleFIelds.yml]:
```suggestion
- name: value
```

**Pull request merged**